### PR TITLE
Change defaults

### PR DESCRIPTION
**Pull request by robbecker-wf**
Change the default formatter and analyzer to the newer options `dart format` and `dart analyze`

### CODE REVIEW

**Comment by aviary2-wf** [comment]
## Security Insights
No security relevant content was detected by automated scans.

## Action Items
* Review PR for [security impact](https://wiki.atl.workiva.net/display/SECURITY/Development+Security+Review+Guidelines); comment "security review required" if needed or unsure
* Verify [`aviary.yaml`](https://wiki.atl.workiva.net/pages/viewpage.action?pageId=42762752#SecurityInsightFramework&Tooling-ConfiguringAviary) coverage of security relevant code
---
_Questions or Comments? Reach out on Slack: #support-infosec._

**Review by evanweible-wf — DISMISSED** [review]

**Review by dustinlessard-wf — DISMISSED** [review]

**Comment by robbecker-wf** [comment]
QA+1 CI Passes

File: lib/src/tools/analyze_tool.dart [hunk 1]
```diff
@@ -74,7 +74,7 @@ class AnalyzeTool extends DevTool {
 
   @override
   FutureOr<int> run([DevToolExecutionContext context]) {
-    useDartAnalyze ??= false;
+    useDartAnalyze ??= true;
```
Comment (todbachman-wf) [inline lib/src/tools/analyze_tool.dart:77]: Maybe we should update the doc comment on `useDartAnalyze` to indicate that we will actually use `dart analyze` by default.

Comment (robbecker-wf) [reply lib/src/tools/analyze_tool.dart:77]: Oh that's a good idea!

**Review by todbachman-wf — APPROVED** [review]

**Review by dustinlessard-wf — CHANGES_REQUESTED** [review]
Looks like the tests are not expecting 'analyze' to be in the buildArgs.

**Comment by robbecker-wf** [comment]
@dustinlessard-wf Indeed. It looks like the analyze args are different depending on if you are using "dart analyze" mode vs "dartanalyzer" mode. Instead of fixing the tests, I think I'll investigate changing the code so that the analyzer args are the same no matter which mode you are in.

**Pull request closed without merging**